src/middleware/header.js

```diff
@@ -223,11 +226,6 @@ module.exports = function (middleware) {
 		], callback);
 	};
 
-	function addTimeagoLocaleScript(scripts, userLang) {
-		var languageCode = utils.userLangToTimeagoCode(userLang);
-		scripts.push({ src: nconf.get('relative_path') + '/assets/vendor/jquery/timeago/locales/jquery.timeago.' + languageCode + '.js' });
-	}
-
 	middleware.renderFooter = function renderFooter(req, res, data, callback) {
```

Comment:
Recommend pulling `res.locals.config.userLang` into a `var userLang = res.locals.config.userLang;` at the beginning of the function, and using that throughout the function instead